modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1776,8 +1788,19 @@ private CompletableFuture<ReplicaResult> processMultiEntryAction(ReadWriteMultiR
                         return completedFuture(new ReplicaResult(result, null));
                     }
 
-                    return validateAtTimestampAndBuildUpdateAllCommand(request, rowIdsToDelete, lastCommitTimes, txCoordinatorId)
-                            .thenCompose(cmd -> applyUpdateAllCommand(cmd, request.skipDelayedAck()))
+                    return validateOperationAgainstSchemaAtTimestamp(request.transactionId())
+                            .thenCompose(
+                                    catalogVersion -> applyUpdateAllCommand(
+                                            rowIdsToDelete,
+                                            lastCommitTimes,
+                                            request.commitPartitionId(),
```

Comment:
Would it make sense to create an overload of `applyUpdateAllCommand` that would accept a `request` and take out commitPartitionId, transactionId, full and skipDelayedAck from it? As this seems to always repeat. This would reduce duplication and also make it easier to handle new message fields in the future (they seem to be added again and again).

Reply:
I've added an overloaded version with request and removed initial one because occurred that there are no usages any longer.